src/Elastic.Markdown/Slices/Layout/_LandingPage.cshtml

```diff
@@ -252,8 +252,8 @@
 					<h3 class="font-sans font-bold text-2xl">Previous versions</h3>
 					<p class="grow mt-4">Browse the docs for previous Elastic product versions.</p>
 					<div class="mt-6">
-						<a href="@Model.Link("/guide")" target="_blank" rel="noopener noreferrer" class="link text-white">
-							View archive docs
+						<a href="https://www.elastic.co/guide" target="_blank" rel="noopener noreferrer" class="link text-white">
```

Comment:
@elastic/docs-engineering I'm not sure if this is correct. It seems `@Model.Link("` substitutes `elastic.co/docs` in. In this case, I don't want to link to a `/docs` page. Is this the correct way to do this? The unfortunate thing is that the link is not environment aware.

Reply:
Full URL seems good.